ivadomed/utils.py

```diff
@@ -69,6 +114,64 @@ def get_pixdim(self, fname):
         px, py, pz = nib_im.header['pixdim'][1:4]
         return px, py, pz
 
+    def remove_small_objects(self, data):
```

Comment:
I feel like the pred_mask should be overwritten with the new post-processing step (save the new nii.gz file)

Reply:
Good Idea!
I could move this function to a `post_processing` module, will do in #151 !